### PR TITLE
New package: rset-2.4

### PR DESCRIPTION
**Pull request by Benjamin-L**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 

- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl (cross)
  - [x] armv7l (cross)
  - [x] armv6l-musl (cross)

### CODE REVIEW

**Comment by Benjamin-L** [comment]
Oh, looks like I never ran the check commands locally, and missed a check dependency on `wget`. My bad!

**Comment by Benjamin-L** [comment]
Alright now I'm *really* not sure what's going on. These checks work fine locally. I'll take a closer look tomorrow when I have time.

**Comment by Benjamin-L** [comment]
Ah, figured it out. This is an upstream problem with `rset` that occurs when running the tests as root. I created an issue for it here: https://github.com/eradman/rset/issues/14. For now I'm going to disable the checks, with a comment to see if the issue is resolved when updating to a newer version later.

File: srcpkgs/rset/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+version=2.1
+revision=1
+build_style=configure
+make_install_args="PREFIX=/usr"
+make_check_target=test
+hostmakedepends="gcc"
+checkdepends="ruby>=2.4 wget"
+short_desc="Configure systems using using any scripting language"
+maintainer="Benjamin Lee <benjamin@typ3.tech>"
+license="GPL-3.0-or-later"
```
Comment (eradman) [inline srcpkgs/rset/template:12]: rset itself (build and installed binaries) uses the ISC license

Comment (Benjamin-L) [reply srcpkgs/rset/template]: Ah, good catch. Fixed

**Comment by Benjamin-L** [comment]
@ericonr can you approve this for a CI run?

**Comment by Benjamin-L** [comment]
I'm updating this package to fix the lint error and bring it to the newest version (2.4), and have discovered that the `webrick` module was [removed][1] from the standard library in version 3.0. This module is required in order to run the tests for `rset`, so I'm going to make a separate PR to add it.

[1]: https://bugs.ruby-lang.org/issues/17303

**Comment by Benjamin-L** [comment]
Alright, I've batched the ruby-webrick package in with rset, since it is a dependency, and added a patch to use `inetutils-hostname` instead of `hostname` in the test, since rset is expecting BSD `hostname`, which supports the `-s` flag.

**Pull request closed without merging**